bool override 						# preempt currently playing pattern
metawear_ros/Vibration[] pattern	# vibration pattern